sats-dna/src/main/kotlin/com/sats/dna/tooling/PreviewAnnotations.kt

```diff
@@ -7,6 +7,7 @@ import androidx.compose.ui.tooling.preview.Preview
 @Preview("Dark Mode", group = "Theme", uiMode = Configuration.UI_MODE_NIGHT_YES)
 annotation class LightDarkPreview
 
-@Preview("Small Font Size", group = "Font Sizes", fontScale = 0.75f)
-@Preview("Large Font Size", group = "Font Sizes", fontScale = 1.50f)
+@Preview("Small Font Size (0.75x)", group = "Font Sizes", fontScale = 0.75f)
+@Preview("Large Font Size (1.50x)", group = "Font Sizes", fontScale = 1.5f)
+@Preview("Huge Font Size (3.0x)", group = "Font Sizes", fontScale = 3f)
```

Comment:
I can't imagine much more than a 3x font scale being usable.

Reply:
Yeah as far as I know it's [just required to support 2x as well](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Text_Size#New_Techniques)?  But I might be misunderstanding what they're trying to say there.